sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,712 @@
+# Azure Communication Chat Service client library for Android
+This package contains the Android SDK for Azure Communication Services for Chat.
+Read more about Azure Communication Services [here](https://docs.microsoft.com/azure/communication-services/overview).
+
+# Getting started
+
+## Prerequisites
+
+* An Azure Communication Resource, learn how to create one from [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource)
+  higher to use this library.
+* The library is written in Java 8. Your application must be built with Android Gradle Plugin 3.0.0 or later, and must
+  be configured to
+  [enable Java 8 language desugaring](https://developer.android.com/studio/write/java8-support.html#supported_features)
+  to use this library. Java 8 language features that require a target API level >21 are not used, nor are any Java 8+
+  APIs that would require the Java 8+ API desugaring provided by Android Gradle plugin 4.0.0.
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this library.
+
+### Versions available
+The current version of this library is **1.0.0-beta.1**.
+
+> Note: The SDK is currently in **beta**. The API surface and feature sets are subject to change at any time before **GA**. We do not currently recommend them for production use.
```

Comment:
Should we clarify what GA stands for?

Reply:
like we discussed offline, we keep this as it is.